core.d.ts

```diff
@@ -16,6 +16,21 @@ declare namespace PinejsClientCoreFactory {
 		[index: string]: T
 	}
 
+	interface MimemessageFactory {
+		factory(cs: BatchFactory): CompiledBatch
+	}
+
+	interface BatchFactory {
```

Comment:
Hmmm the option word is already used in the context of 'ODataOptions` and it might be slightly confusing. `BatchParams` sounds like a good compromise to me

Reply:
Sounds equivalently appropriate :+1: